variables.tf

```diff
@@ -335,62 +337,76 @@ variable "peerings" {
   }))
   default     = {}
   description = <<DESCRIPTION
-    (Optional) A map of association objects to create peerings between the created circuit and the designated gateways. 
+(Optional) A map of association objects to create peerings between the created circuit and the designated gateways. 
+
+- `peering_type` - (Required) The type of peering. Possible values are `AzurePrivatePeering`, `AzurePublicPeering`, and `MicrosoftPeering`.
+- `vlan_id` - (Required) The VLAN ID for the peering.
+- `primary_peer_address_prefix` - (Optional) The primary peer address prefix.
+- `secondary_peer_address_prefix` - (Optional) The secondary peer address prefix.
+- `ipv4_enabled` - (Optional) Is IPv4 enabled for this peering. Defaults to `true`.
+- `shared_key` - (Optional) The shared key for the peering.
+- `peer_asn` - (Optional) The peer ASN.
+- `route_filter_id` - (Optional) The ID of the route filter to associate with the peering.
```

Comment:
@jchancellor-ms Please update this to route_filter_resource_id as well.

Reply:
Updated